src/screens/Asl/components/AslTable/index.js

```diff
@@ -110,7 +188,7 @@ const AslTable = props => {
       const hostName = window.location.hostname;
       if (hostName !== '') {
         if (source === 'ASLCORE') {
-          setVideoUrl(`https://${hostName}/data/aslvideos/aslcore/original/${uniqueASLIdentifier}.mp4`);
+          setVideoUrl(`https://ct-dev.ncsa.illinois.edu/data/aslvideos/aslcore/original/${uniqueASLIdentifier}.mp4`);
```

Comment:
No hard coded server names. The frontend is designed to work with arbitrary backend servers

Reply:
This part is back to ${hostname} for the latest version of asl-gallery-1.0 I think. It might be commit mistake.